### PR TITLE
Makes slight changes to MultiNic docs

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-13674

Link to docs preview:
https://61907--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/overriding-default-node-ip-selection-logic.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ NODEIP_HINT=192.0.2.1
 +
 [source,terminal]
 ----
-$ echo 'NODEIP_HINT=192.0.2.1' | base64
+$ echo -n 'NODEIP_HINT=192.0.2.1' | base64
```
Comment (rbbratta) [inline modules/overriding-default-node-ip-selection-logic.adoc:36]: `base64 -w0` to disable line wrapping

**Review by cybertron — COMMENTED** [review]
/lgtm

Adding -w0 would be a good improvement too, but the changes lgtm.

File: modules/overriding-default-node-ip-selection-logic.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/overriding-default-node-ip-selection-logic.adoc]: @rbbratta fixed. PTAL when you get a moment. 

 Thanks a lot!

**Comment by rbbratta** [comment]
YAML verified.

```
yq  '.spec.config.storage.files[] | select(.path == "/etc/default/nodeip-configuration") | .contents.source' < test.yaml | python3 -c 'import sys ; from urllib.request import urlopen ;  sys.stdout.buffer.write(urlopen(sys.stdin.read()).read())'
```

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**